Formula/openjfx.rb

```diff
@@ -0,0 +1,89 @@
+# For `pour_bottle? do` block in main formula body below:
+class XcodeCLTRequirement < Requirement
+  fatal true
+
+  satisfy(:build_env => false) do
+    MacOS::CLT.installed?
+  end
+
+  def message
+    <<~EOS
+      This software requires the Xcode CLT in order to build.
+    EOS
+  end
+end
+
+class Openjfx < Formula
+  desc "Open-source, next-generation Java client application platform."
+  homepage "https://openjfx.io/"
+  url "http://hg.openjdk.java.net/openjfx/jfx-dev/rt", :revision => "22c69e868654", :using => :hg
+  sha256 "0be0d342dec0c8ca777ddce3e7ab2afa1e4145181ece4dcaec9f6385cf728fbb"
+  mirror "http://hg.openjdk.java.net/openjfx/jfx-dev/rt/archive/22c69e868654.tar.gz"
```

Comment:
Please use HTTPS for `url` and `mirror` as well.

Reply:
@vszakats:  

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;I don't know if HTTPS versions of those downloads are available yet, but I'll check for them.